conf/messages

```diff
@@ -707,6 +707,7 @@ declaration.p.3=Once you submit the form you will not be able to go back and mak
 label.declaration=Declaration
 declaration.iAgree=I agree to the above declaration
 declaration.required=You must agree to the declaration to continue
+declaration.file.receipt=Awaiting evidence receipt, please try again in a moment.
```

Comment:
I don't think this makes it any clearer for the user... I wouldn't know what that means. I tick a box to declare something, press submit, and back to the same page with "Awaiting evidence receipt ..." I'd be well confused. Has the text come from Karen?

Reply:
It is clearer. But could be worded better.